Snowflake/Snowflake_Ingest_csv_data_from_local_stage.ipynb

```diff
@@ -0,0 +1,598 @@
+       " 'statement': \"CREATE OR REPLACE FILE FORMAT my_csv_format TYPE = CSV FIELD_DELIMITER = ',' SKIP_HEADER = 1\"}"
+      ]
+     },
+     "execution_count": 9,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "results_ff = snowflake.file_format.create(\n",
+    "    'my_csv_format', \n",
+    "    'CSV', \n",
+    "    or_replace=True,\n",
+    "    FIELD_DELIMITER=\"','\",\n",
+    "    SKIP_HEADER=1\n",
+    ")\n",
+    "\n",
+    "results_ff"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Creating file format with wrong parameter (not applicable for JSON file format, see [docs](https://docs.snowflake.com/en/sql-reference/sql/create-file-format.html))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Something went wrong!\n",
+      "002135 (42601): SQL compilation error:\n",
+      "Option ENABLE_OCTAL is not valid for file format type CSV.\n"
+     ]
+    }
+   ],
+   "source": [
+    "try:\n",
+    "    snowflake.file_format.create('my_invalid_csv_format', 'CSV', or_replace=True, ENABLE_OCTAL = \"TRUE\")\n",
+    "except ProgrammingError as pe:\n",
+    "    print('Something went wrong!')\n",
+    "    print(pe)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Dropping file format"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# snowflake.file_format.drop('my_csv_format', if_exists=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Creating internal stage"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'results': [('Stage area MY_INTERNAL_STAGE successfully created.',)],\n",
+       " 'description': [ResultMetadata(name='status', type_code=2, display_size=None, internal_size=16777216, precision=None, scale=None, is_nullable=True)],\n",
+       " 'statement': 'CREATE OR REPLACE STAGE my_internal_stage FILE_FORMAT = my_csv_format'}"
+      ]
+     },
+     "execution_count": 12,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "snowflake.stage.create(\n",
+    "    stage_name='my_internal_stage', \n",
+    "    or_replace=True,\n",
+    "    file_format_name='my_csv_format'\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Putting data on the stage"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "snowflake.stage.put(\n",
+    "    filepath='file://~/Downloads/sales_data.csv',\n",
```

Comment:
`filepath` value should be `file://sales_data.csv` instead of `file://~/Downloads/sales_data.csv`

Reply:
Actually, I did that on purpose, because I wanted to show how to put a relative path, instead to working directory. 

Do you think we should change that? Because the location of the file always will depend on the location when somebody stores it locally